rebuild.sh

```diff
@@ -0,0 +1,4 @@
+jlpm
+jlpm run build
+pip install . --upgrade
+jupyter lab
```

Comment:
please remove this

Reply:
Done, see: https://github.com/deathbeds/jupyterlab-starters/pull/78